release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1108,7 +1108,7 @@ They correspond to older, similarly named SCCs, but with the following enhanceme
 * `seccompProfile` is defaulted to `runtime/default` if unset. In previous releases, this field had to be empty.
 * `allowPrivilegeEscalation` must be unset or set to `false` in security contexts. Previously, a `true` value was allowed.
 
-New clusters allow use of the `restricted-v2` SCC for any authenticated user in place of the `restricted` SCC; the `restricted` SCC is no longer available to users of new clusters, unless the access is explicitly granted. Clusters that are upgraded to {product-title} 4.11 from previous versions allow both `restricted-v2` and `restricted` SCCs.
+In {product-title} 4.11, the `restricted-v2` SCC is now the default SCC for new installations, instead of the `restricted` SCC. In new clusters, the `restricted-v2` SCC is used for any authenticated user in place of the `restricted` SCC. The `restricted` SCC is no longer available to users of new clusters, unless the access is explicitly granted. In clusters originally installed in {product-title} 4.10 or earlier, all authenticated users can use the `restricted` SCC when upgrading to {product-title} 4.11 and later. This keeps the default security permissions for {product-title} as secure-by-default and aligns with pod security admission and pod security standards from the upstream Kubernetes project, while keeping the permissions of upgraded clusters consistent with their previous state.
```

Comment:
It reads like "`restricted-v2` SCC is the default SCC ..., instead of the `restricted` SCC". This makes others at first glance get: `restricted` SCC is not a default SCC. Current doc mentions "default SCCs", anyuid hostaccess ... privileged restricted restricted-v2 all are default SCCs, each SCC is a default SCC.
s/`restricted-v2` SCC is now the default SCC/`restricted-v2` SCC is now the default granted SCC available to users/

Reply:
Thanks @xingxingxia! Updated with this change, let me know if it looks better now.